resources/i18n/pl/cosmic_comp.ftl

```diff
@@ -1,20 +1,20 @@
 grow-window = Powiększ
 shrink-window = Zmniejsz
 swap-windows = Zamień Okna
-stack-windows = Zakrywające Okna
+stack-windows = Okna w Kartach
 unknown-keybinding = <unset>
 window-menu-minimize = Minimalizuj
 window-menu-maximize = Maksymalizuj
-window-menu-tiled = Float window
+window-menu-tiled = Pływające Okno
 window-menu-screenshot = Zrób zrzut ekranu
 window-menu-move = Przesuń
 window-menu-resize = Zmień rozmiar
 window-menu-move-prev-workspace = Przenieś do poprzedniego obszaru roboczego
 window-menu-move-next-workspace = Przenieś do następnego obszaru roboczego
-window-menu-stack = Włącz zakrywanie okien
-window-menu-unstack-all = Usuń zakrywanie okien
-window-menu-unstack = Usuń zakrycie okna
-window-menu-sticky = Przylepne okno
+window-menu-stack = Stwórz okna w kartach
+window-menu-unstack-all = Rozdziel okna w kartach
```

Comment:
Rozgrupuj okna

Reply:
I'm not too sure about this one, perhaps Rozdziel grupę okien would feel better?

Reply:
Sure